test/functional/Java8andUp/src/org/openj9/test/attachAPI/AttachApiTest.java

```diff
@@ -200,6 +203,8 @@ protected boolean checkLoadAgentOutput(String errOutput,
 		ArrayList<String> cmdLine = new ArrayList<>();
 		cmdLine.add(commandName);
 		cmdLine.addAll(args);
+		Optional<String> commandLineString = cmdLine.stream().reduce((s1, s2) -> s1 + " " + s2);
+		log("command line: " + commandLineString.get());
```

Comment:
`String.join()` is what you want here.
```
log("command line: " + String.join(" ", cmdLine));
```
Alternatively, there's the collector: `joining(String delimiter, String prefix, String suffix)`
```
log(cmdLine.stream().collect(Collectors.joining(" ", "command line: ", "")));
```

Reply:
Thanks for the suggestion.  Done.